deployment/armada-server/charts/scheduler-ingester/values.yaml

```diff
@@ -0,0 +1,9 @@
+# Default values for scheduleringester.
+metricsPort: 9000
+replicas: 1
+postgres:
```

Comment:
Default values that don't work seem odd to me?  Not a big deal though but it sounds like if someone used this it would fail.